pkg/volume/azure_dd/azure_common.go

```diff
@@ -201,3 +205,25 @@ func strFirstLetterToUpper(str string) string {
 	}
 	return libstrings.ToUpper(string(str[0])) + str[1:]
 }
+
+// getDiskLUN : deviceInfo could be a LUN number or a device path, e.g. /dev/disk/azure/scsi1/lun2
+func getDiskLUN(deviceInfo string) (int32, error) {
+	var diskLUN string
+	if len(deviceInfo) <= 2 {
+		diskLUN = deviceInfo
```

Comment:
Check whether it is a number or not?

Reply:
Is there any possibility of supporting lun number >= 100 in the future?

Reply:
it's checking under below:
```
lun, err := strconv.Atoi(diskLUN)
```
Azure platform only support up to 64 disk number